AutoRest/TestServer/swagger/azure-special-properties.json

```diff
@@ -732,7 +732,52 @@
           }
         }
       }
-    },    
+    },
+    "/azurespecials/odata/filter": {
+      "get": {
+        "operationId": "odata_getWithFilter",
+        "description": "Specify filter parameter with value '$filter=id gt 5 and name eq \'foo\'&$orderby=id&$top=10'",
```

Comment:
This is wrong according to swagger schema validation on https://json-schema-validator.herokuapp.com/ .